src/components/SettingsCheckbox/SettingsCheckbox.vue

```diff
@@ -0,0 +1,150 @@
+<!--
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  - @author Greta Doci <gretadoci@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<docs>
+## One checkbox
+
+```vue
+	<template>
+		<div>
+			<b-form-checkbox
```

Comment:
What is b-form-checkbox?

Reply:
For cross browser consistency,  <b-form-checkbox> use Bootstrap's custom checkbox input to replace the browser default checkbox input. 
Should I keep it simple: 
<input
  type="checkbox"> ?